crutest/src/main.rs

```diff
@@ -1045,7 +1035,7 @@ async fn main() -> Result<()> {
      * Build the region info struct that all the tests will use.
      * This includes importing and verifying from a write log, if requested.
      */
-    let mut region_info = match get_region_info(&block_io).await {
+    let mut region_info = match get_region_info(&block_io, is_encrypted).await {
```

Comment:
Pre-existing, but this could be written as
```rust
    let mut region_info = get_region_info(&block_io, is_encrypted)
        .await
        .context("failed to get region info")?;
```
(instead of a `match`; requires importing `anyhow::Context`)